From 19e48a6eee20d5f34f79a3b8d4e1c694169c1d7b Mon Sep 17 00:00:00 2001
From: Fabio M. Di Nitto <fdinitto@redhat.com>
Date: Sun, 23 Oct 2016 08:23:52 +0200
Subject: [PATCH 01/18] [build] Fix build on RHEL7.3 latest

header inclusion have changed

Signed-off-by: Fabio M. Di Nitto <fdinitto@redhat.com>
(cherry picked from commit e204e871ec4eb705d177598d5c7ae8f0446a280f)
---
 lib/sam.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/lib/sam.c b/lib/sam.c
index 1596ae2..33aa394 100644
--- a/lib/sam.c
+++ b/lib/sam.c
@@ -46,6 +46,7 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <errno.h>
+#include <poll.h>
 
 #include <corosync/corotypes.h>
 #include <qb/qbipcc.h>
-- 
1.7.1

